os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -193,6 +196,24 @@ static int binary_manager_load(int bin_idx)
 		if (!binp)
 #endif
 		{
+#ifdef CONFIG_BINARY_SIGNING
+			/* Check signature */
+			ret = up_verify_usersignature(BIN_PARTADDR(bin_idx, (BIN_USEIDX(bin_idx))));
+			if (ret == OK) {
+				lldbg("%s Signature Checking Success\n", BIN_NAME(bin_idx));
```

Comment:
It is better to use `printf` instead of `lldbg`, because of preventing mixed logs.